spec/system/jobseekers/jobseekers_can_search_for_jobs_spec.rb

```diff
@@ -204,7 +204,8 @@
         click_on I18n.t("buttons.search")
       end
 
-      it "shows distance between school and their location" do
+      # These three tests all depend on the same location search. Could consider combining them into a single example to reduce page visits and searches and improve test speed.
+      it "shows distance between school and their location", js: true do
```

Comment:
Have combined these 3 tests into one. Each one involved relied on entering the same details and running the location search, so I think the efficiency gains will make up for any loss of granularity in our tests.